src/tasks/WasmBuildTasks/GenerateAOTProps.cs

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
this should ideally be added to https://github.com/dotnet/runtime/blob/d08f7f2074bf3c553910f4ffaf67fb008a151aef/src/tasks/tasks.proj#L8-L11

Reply:
I'm not sure I understand. This file will get built with `WasmBuildTasks`. What should be added to `tasks.proj`?